matchbox_socket/src/ggrs_socket.rs

```diff
@@ -1,13 +1,26 @@
 use ggrs::{Message, PlayerType};
 use matchbox_protocol::PeerId;
 
-use crate::WebRtcSocket;
+use crate::{ChannelConfig, MessageLoopFuture, WebRtcSocket, WebRtcSocketBuilder};
 
 #[derive(Debug, thiserror::Error)]
 #[error("The client has not yet been given a Peer Id")]
 pub struct UnknownPeerId;
 
 impl WebRtcSocket {
+    /// Creates a [`WebRtcSocket`] and the corresponding [`MessageLoopFuture`] for a
+    /// socket with a single channel configured correctly for usage with GGRS.
+    ///
+    /// The returned [`MessageLoopFuture`] should be awaited in order for messages to
+    /// be sent and received.
+    ///
+    /// Please use the [`WebRtcSocketBuilder`] to create non-trivial sockets.
+    pub fn new_ggrs(room_url: impl Into<String>) -> (WebRtcSocket, MessageLoopFuture) {
```

Comment:
Shouldn't this be a feature method?

Reply:
See below. We could perhaps document that you need to enable the feature, though...